frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -681,9 +724,72 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
             />
           ) : null}
 
+          {this.state.ongoingUploads.length > 0 && (
+            <Alert
+              message={
+                <>
+                  Ongoing unfinished Dataset Uploads{" "}
+                  <Tooltip
+                    title="This list shows all uploads from the past two weeks that were started by you but not completed. You can try continuing these uploads."
+                    placement="right"
+                  >
+                    <InfoCircleOutlined />
+                  </Tooltip>
+                </>
+              }
+              description={
+                <div
+                  style={{
+                    paddingTop: 8,
+                  }}
+                >
+                  {this.state.ongoingUploads.map((ongoingUpload) => (
+                    <Row key={ongoingUpload.uploadId} gutter={16}>
+                      <Col span={8}>{ongoingUpload.dataSourceId.name}</Col>
+                      <Col span={8}>
+                        <Button
+                          type="link"
+                          onClick={() => {
+                            const currentFormRef = this.formRef.current;
+                            if (!currentFormRef) {
+                              return;
+                            }
+                            currentFormRef.setFieldsValue({
+                              name: ongoingUpload.dataSourceId.name,
+                              targetFolderId: ongoingUpload.folderId,
+                              initialTeams: this.state.possibleTeams.filter((team) =>
+                                ongoingUpload.allowedTeams.includes(team.id),
+                              ),
+                            });
+                            this.setState({
+                              uploadId: ongoingUpload.uploadId,
+                              continuingOldUpload: true,
+                            });
+                            Toast.info(
+                              "To continue the selected Upload please make sure to select the same file(s) as before. Else unexpected behavior may occur.",
```

Comment:
```suggestion
                              "To continue the selected upload please make sure to select the same file(s) as before. Otherwise, the upload may be corrupted.",
```